src/data.js

```diff
@@ -0,0 +1,86 @@
+export default [
+  {
+    name: 'Contact 0',
+    phoneNumber: '+37529xxxxx00',
```

Comment:
xxx is not needed, cuz you will be add contact as a number so full phone number should be displayed

Reply:
Replaced xxx with digits